coderd/workspaceagents.go

```diff
@@ -1603,97 +1645,57 @@ func (api *API) watchWorkspaceAgentMetadata(rw http.ResponseWriter, r *http.Requ
 	}
 	// Prevent handler from returning until the sender is closed.
 	defer func() {
-		<-senderClosed
+		<-sseSenderClosed
 	}()
 
-	const refreshInterval = time.Second * 5
-	refreshTicker := time.NewTicker(refreshInterval)
-	defer refreshTicker.Stop()
+	// We send updates at most every second.
+	const sendInterval = time.Second * 1
+	sendTicker := time.NewTicker(sendInterval)
```

Comment:
A `NewTimer` might be better, coupled with `Reset` if we want to be literal with the comment above (at most every second). The next ticker could immediately be ready depending on how things went.

Reply:
Exactly every second was my intention — updated the comment.